assets/src/scss/layout/_navbar.scss

```diff
@@ -709,14 +727,15 @@ $navbar-default-height: 60px;
   }
 
   @include large-and-up {
+    --country-list-- {
+      height: 344px;
+      width: 80%;
+      left: 10%;
+    }
+    max-height: calc(100vh - #{$menu-height-large});
     position: absolute;
-    height: 344px;
-    width: 80%;
-    left: 10%;
     overflow-x: hidden;
     overflow-y: hidden !important;
-    max-height: 344px;
```

Comment:
I added `height` since it seems useful to try different values for this, especially if other properties are changed that affect the height needed. I removed `max-height` because otherwise you'd have to change both of them, and we set them to the same value anyway.